batch/batch/server/server.py

```diff
@@ -216,56 +214,36 @@ def _has_next_task(self):
                         }),
             spec=pod_spec)
 
-        try:
-            pod = v1.create_namespaced_pod(
-                HAIL_POD_NAMESPACE,
-                pod_template,
-                _request_timeout=KUBERNETES_TIMEOUT_IN_SECONDS)
-            self._pod_name = pod.metadata.name
-
-            await db.jobs.update_record(self.id,
-                                        pod_name=self._pod_name)
-
-            log.info('created pod name: {} for job {}, task {}'.format(self._pod_name,
-                                                                       self.id,
-                                                                       self._current_task.name))
-        except kube.client.rest.ApiException as err:
+        pod, err = await app['k8s'].create_pod(body=pod_template)
+        if err is not None:
+            traceback.print_tb(err.__traceback__)
```

Comment:
What is the traceback for? Isn't it sufficient in the log statement below?

Reply:
The log statement below doesn't print a traceback, just the error message.